style.css

```diff
@@ -0,0 +1,147 @@
+*{
+    padding: 0;
+    margin: 0;
+    box-sizing: border-box;
+}
+
+.wrapper{
+    margin: 100px auto;
+    width: 320px;
+    background-color: #000;
+    position: relative;
+    color: white;
+    text-align: center;
+    height: 40px;
+    font-family: 'Arial';
+}
+
+#check-menu{
+    display: none;
+}
+
+label{
+    display: block;
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    cursor: pointer;
+    z-index: 2;
+    line-height: 40px;
+}
+
+.burger-line{
+    position: absolute;
+    top: 8px;
+    right: 10px;
+    width: 35px;
+    height: 2px;
+    background-color: #fff;
+    transition: .5s all;
+}
+
+.second, .third{
+    top: 18px;
+}
+
+.fourth{
+    top: 28px;
+}
+
+.main-menu{
+    display: block;
+    position: absolute;
+    width: 100%;
+    bottom: 0;
+    left: 0;
+    transform: translateY(100%);
+    max-height: 0;
+    font-size: 0;
+    overflow: hidden;
+    transition: .5s all;
+}
+
+a{
+    display: block;
+    width: 100%;
+    padding: 10px;
+    color: white;
+    text-decoration: none;
+    border-top: 1px solid rgba(255,255,255,.3);
+    background-color: #000;
+}
+
+a:hover{
+    background-color: rgba(0,0,0,.9);
+}
+
+#check-menu:checked ~ .main-menu{
+    max-height: 500px;
+    font-size: 16px;
+}
+
+#check-menu:checked ~ .first{
+    display: none;
+}
+
+#check-menu:checked ~ .second{
+    transform: rotate(45deg);
+}
+
+#check-menu:checked ~ .third{
+    transform: rotate(-45deg);
+}
+
+#check-menu:checked ~ .fourth{
+    display: none;
+}
+
+.title {
+  font-size: 10px;
+}
+
+@media all and (max-width: 800px) {
+.main-menu { justify-content: space-around; }
+.main-menu a{ flex-grow:1; }
+}
+
+@media all and (max-width: 500px) {
+.main-menu { flex-direction: column; }
+}
+
+@media (min-width: 320px) {
+.title {
+    font-size: 15px;
+	}
+}
+
+@media (min-width: 481px) {
+.title {
+    font-size: 25px;
+	}
+}
+
+@media (min-width: 641px) {
+.title {
+    font-size: 35px;
+	}
+}
+
+@media (min-width: 961px) {
+.title {
+    font-size: 45px;
+	}
+}
+
+@media (min-width: 1025px) {
+.title {
+    font-size: 55px;
+	}
+}
+
+@media (min-width: 1281px) {
+.title {
+    font-size: 65px;
+	  }
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/ad/pullRequest?prid=5498083)